pkg/commands/git_commands/file.go

```diff
@@ -62,7 +62,7 @@ func (self *FileCommands) GetEditCmdStrLegacy(filename string, lineNumber int) (
 	editCmdTemplate := self.UserConfig.OS.EditCommandTemplate
 	if len(editCmdTemplate) == 0 {
 		switch editor {
-		case "emacs", "nano", "vi", "vim", "nvim":
+		case "emacs", "micro", "nano", "vi", "vim", "nvim":
```

Comment:
You can drop this commit again, it's not worth updating this. It's the legacy support for an older version of the config entries; we're going to remove this at some point.